docs/get-started/v2/get-started/features/recordings/storage.mdx

```diff
@@ -59,44 +65,54 @@ You can configure the **`type`** field of recording object to `s3` for AWS, `oss
 - Region: Name of the region where your bucket is located in
 - Prefix for Upload Path: Define the directory name (optional)
 
+</Tab>
+
 ## Fetching the asset
 
-### Using the 100ms Dashboard
+<Tabs id="fetch-recordings" items={['Dashboard','REST API','Webhooks']} />
+
+<Tab id='fetch-recordings-0'>
 
 You can access your recordings on the [sessions page](https://dashboard.100ms.live/sessions) in the 100ms Dashboard.
 
 ![Recording Links](/docs/v2/recording-links-session.png)
 
-### Using the REST API
+</Tab>
+
+<Tab id='fetch-recordings-1'>
 
 Recordings generate [recording assets](/server-side/v2/api-reference/recording-assets/overview) that can be fetched with the REST API.
 
-### Get webhooks
+</Tab>
+
+<Tab id='fetch-recordings-2'>
 
 100ms can send webhooks when the recording has stopped and is available for download. The recording path is available in following webhook responses:
 
-- Room composite (also called browser) recording: [beam.recording.success](/server-side/v2/introduction/webhook#beamrecordingsuccess) (attribute: `recording_path`)
-- Room composite - legacy (also called SFU) recording: [recording.success](/server-side/v2/introduction/webhook#sfu-recording-events) (attribute: `recording_path`)
-- Recordings enabled with live streaming (HLS): [hls.recording.success](/server-side/v2/introduction/webhook#hlsrecordingsuccess)
+- Room Composite (also called browser) recording: [beam.recording.success](/server-side/v2/introduction/webhook#beamrecordingsuccess) (attribute: `recording_path`)
+- Room Composite - Legacy (also called SFU) recording: [recording.success](/server-side/v2/introduction/webhook#sfu-recording-events) (attribute: `recording_path`)
+- Live Stream Recording (HLS): [hls.recording.success](/server-side/v2/introduction/webhook#hlsrecordingsuccess)
+
+</Tab>
 
 ### Path formats
 
 The recording path for these respective recordings will look like:
 
-1. Room composite (also called browser) recording: `s3://<location>/<prefix>/beam/<room_id>/<start_date>/Rec-<room_id>-<epoch>.mp4`
+1. Room Composite (also called browser) recording: `s3://<location>/<prefix>/beam/<room_id>/<start_date>/Rec-<room_id>-<epoch>.mp4`
 
-2. Room composite - legacy (also called SFU) recording:
+2. Room Composite - Legacy (also called SFU) recording:
 
     1. Composite: `s3://<location>/<prefix>/<room_id>/<start_date>/<session_id>/Rec-<session_id>-<epoch>.mp4`
 
     2. Individual: `s3://<location>/<prefix>/<room_id>/<start_date>/<session_id>/<peer_id>/<stream_id>/<track_id>.webm`
 
 3. Multi-resolution recording (available in HLS): `s3://<location>/<prefix>/hls/<room_id>/<start_date>/<epoch>/file-recording/Rec-<room_id>-<epoch>-<layer_index>.mp4`
 
-4. VOD recording (available in HLS): `s3://<location>/<prefix>/hls/<room_id>/<start_date>/<epoch>/vod/Rec-<room_id>-<epoch>.zip`
+4. VOD recording (Zipped - available in HLS): `s3://<location>/<prefix>/hls/<room_id>/<start_date>/<epoch>/vod/Rec-<room_id>-<epoch>.zip`
```

Comment:
before this para .. the name was different

Reply:
I'll correct this to:
HLS Recording (Zipped VoD format)